dashboard/app/controllers/admin_users_controller.rb

```diff
@@ -101,33 +101,59 @@ def manual_pass
     redirect_to :manual_pass_form
   end
 
+  # display the user search form, users found, and permissions for first user found
+  # grant / revoke permissions actions redirect back to this form to display the updated permission list
   def permissions_form
+    @users = Array.new # list of users that match search
+    @permissions = Array.new
```

Comment:
You can use [User.none](https://apidock.com/rails/v4.0.2/ActiveRecord/QueryMethods/none) to construct an empty relation.